src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Managed.cs

```diff
@@ -13,6 +13,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 
+#pragma warning disable CA1416 // Validate platform compatibility, using APIs not supported on browser, but safe to suppress
```

Comment:
Can you exclude the file from browser build instead?

Reply:
It is excluded from browser build, but was not able to exclude from enabling the browser warnings.
We have `'$(TargetsBrowser)' == 'true' or '$(TargetsAnyOS)' == 'true'` condition for enabling browser warning, but the project has browser and multitargeted (`$(NetCoreAppCurrent);$(NetCoreAppCurrent)-Browser`) TFMs, the file is not included in browser build with `'$(TargetsBrowser)' != 'true'` condition but the warnings will be enabled for multitargeted build by the  '$(TargetsAnyOS)' == 'true' condition even it will not be included in browser build. I was not able to solve this with build conditions, suggestions appreciated cc @safern @ViktorHofer

Reply:
cc @ericstj

Reply:
> We have '$(TargetsBrowser)' == 'true' or '$(TargetsAnyOS)' == 'true' condition for enabling browser warning

What if instead you used the configuration system to trigger this? You could enable the browser warning based on when the config being built was the "best" configuration from browser.  This is how we enabled trimming, eg: https://github.com/dotnet/runtime/blob/05ab615553d49a21b66eac71ea94d5578e63f124/src/libraries/Directory.Build.targets#L91

So you could have something like:
```XML
    <BinPlaceTargetFrameworks Include="$(NetCoreAppCurrent)-Browser" SetProperties="EnableBrowserWarnings=true" />
```
The BinPlaceTargetFrameworks item name is a bit of a misnomer, that can be used without actually binplacing anything I think.  @Anipik

Reply:
I had no luck with using this configuration, even couldn't make work simpler, browser only case, some rules i tried:
```xml
<BinPlaceTargetFrameworks Include="$(NetCoreAppCurrent)-Browser" SetProperties="EnableBrowserWarnings=true" />

<BinPlaceTargetFrameworks Include="$(NetCoreAppCurrent)-$(TargetOS)" Condition="'$(TargetOS)' == 'Browser'" SetProperties="EnableBrowserWarnings=true" />

<BinPlaceTargetFrameworks Include="$(NetCoreAppCurrent)-Browser">
  <SetProperties Condition='$(EnableBrowserWarnings)' == ''">EnableBrowserWarnings=true</SetProperties>
</BinPlaceTargetFrameworks>

<BinPlaceTargetFrameworks Include="$(NetCoreAppCurrent)-$(TargetOS)" Condition="'$(TargetOS)' == 'Browser'">
  <SetProperties>EnableBrowserWarnings=true</SetProperties>
</BinPlaceTargetFrameworks>
 ```
Some of it even didn't build, even they built the value is not set, i tried to use them in `versioning.targets`:
```xml
  <ItemGroup Condition="'$(EnableBrowserWarnings)' == 'true'">
    <SupportedPlatform Include="browser"/>
  </ItemGroup>
``` 
Didn't work, tried to use in the same file (`Directory.Build.targets`) also didn't work

Reply:
Thanks @Anipik, this is resolved now